src/3d/qgs3dutils.h

```diff
@@ -36,13 +36,16 @@ namespace Qt3DExtras
 #include "qgs3dtypes.h"
 #include "qgsaabb.h"
 #include "qgsray3d.h"
+#include "qgsraycastingutils_p.h"
```

Comment:
@uclaros this breaks using qgs3dutils.h, see https://github.com/qgis/QGIS/issues/57294